### ISSUE TITLE
parser: support SQL comments

### ISSUE THREAD

**Issue by TimSatke** [tomarrell/lbadd#200]
Extend the parser so that he supports comments as specified here (https://www.sqlite.org/lang_comment.html).

**Comment by Shivam010** [comment]
Hello,
I would like to grab this issue and work on it if no one is working on it.

**Comment by tsatke** [comment]
Hi, please move to xqueries/xdb and find that issue. We no longer work in this repo

**Comment by tomarrell** [comment]
You're more than welcome to pick up this issue here if you'd like.

Tim has made a fork and has moved to that, however you're welcome to contribute to whichever you would like.

I'd be happy to review a PR if you open it.

**Comment by Shivam010** [comment]
Ok, but is there any particular reason for maintaining two different project works?
I am kinda confused with the parting. 
Can I know the goal set or the differences between both the projects if possible, @tomarrell @tsatke?

**Comment by Shivam010** [comment]
I would really like to work on building a database. I like the project.

**Comment by tsatke** [comment]
@tomarrell is inactive since about febuary, and I'm not an admin in this repo, so I can't archive it or anything

**Comment by tsatke** [comment]
the goal set of the new project is just a different roadmap really

**Comment by tsatke** [comment]
Let's take this discussion to xqueries/xdb#1

**Comment by tomarrell** [comment]
Sure, I have certainly been less than active, that is true. However, I would not consider this project abandoned by any means. I would consider it more of a _long running project_.

The goal of this project when I first embarked on it was to build something that was ultimately as simple as possible in order to act as a good reference for people who want to learn the inner workings of a distributed database.

If you share the same goals, then you're more than welcome to contribute here if you would like.

Otherwise, xqueries/xdb looks to be a good option if you would like to take it past this above goal, as it appears they're looking to build something a little more tuned for production use.

**Comment by tsatke** [comment]
Yes, I would fully agree with that description! Feel free to choose either one, you're welcome in both projects!

**Comment by Shivam010** [comment]
Right now I would like to choose xdb. No hard feelings.

**Comment by tomarrell** [comment]
No worries at all. Best of luck!
